### ISSUE TITLE
[Web] Revisit classroom callout on the community library page.

### ISSUE THREAD

**Issue by seanlip** [oppia/design-team#88]
**Platform**
Oppia Web

**Describe the request**

On the community library page at https://www.oppia.org/community-library, there is a callout to the math classroom.

![Screenshot from 2023-07-26 00-40-23](https://github.com/oppia/design-team/assets/10575562/0fc7d905-ab26-4ad8-8acd-dbf02d0a5919)

I'm not sure if the copy for this is quite right: it uses the word "course" (which I'm not sure we're using) and the second "sentence" is really a sentence fragment. So, filing this issue to get a UXW audit.

**Context for the request**

This is an existing feature that is for learners who are browsing the community library page.

**DESIGN APPROVAL CHECKLIST -- DO NOT EDIT.**

- [ ] UX Design approved (Android mocks should include dark mode)
- [x] UX Writing approved
- [ ] Design Systems Team approved
- [ ] Product approved

_For a design project to be completed, it needs approval from UXD, UXW, DS and PM. Please tag the following reviewers when your project is ready (at least one reviewer for each of the 4 bullet points below):_
- UXD: @rflore / @S4v8n (for Web projects)
- UXW: @stepmalt @davishedrick 
- Design systems: @qluuu / @celestinetales
- PM: @seanlip

**Comment by seanlip** [comment]
Note: It is totally fine if the result of this project ends up being "no changes are needed". I just wanted to put it up for review since I don't think it's been looked at carefully from a design perspective yet.

**Comment by stepmalt** [comment]
Notes from the Community Library page, displaying all of the courses:

There’s a period missing after “magnetism” under the Electromagnetism lesson. This error repeats at the end of :  Pitch Perfect, What Is a Fraction, The Meaning of “Equal Parts”,  Welcome To Oppia, 1-2 Intro to Python, 1-3 Installing Python, Hygiene and Disease, Matter, Get Electrified -1, Tree Identifying, vegetable egg omelette(which needs to be capitalized), HelloFraction(is this supposed to be one word?), English prepositions (that should be capitalized), They’re there, and their (should also be capitalized, Let’s Have Fun With Grammar, Overpopulation in India,  Indian Contract law (law should be capitalized), Safeguarding training, Forms of Business org (also, why is this in all caps?).
 All of the above require a period at the end of the sentence describing the lesson.

Maths Quiz 2 - there’s an extra space between the last word and the exclamation point, also I think we should edit this copy here.

What are the Place Values — I wonder if we should remove ‘the’ from this?

Also, I think it would be more visually pleasing if we just moved to the next line when words are cut off. For example Pro-gramming, since it goes onto the next line. I think we should omit these and just move to the next line when we can. It looks much better than numerous broken up words throughout the lessons. 

Also, we need to be consistent with the use of & or And —  one needs to be chosen, and we need to to stick with this throughout all of the lesson titles.

“From” should be capitalized in Physics From Light

I am confused here about part 1. For example, Scientific Principles 1 — is this part 1 in the series? But then, I also see Concept of diseases (part 1). Are these both meant to represent part 1 or is it different? Also, diseases should be capitalized.  Also there is not consistency with how part is written, for example, “Carla Listens, Part II” and “Concept of diseases (part 1)”. Those are two different stylings.

This is some of what I noticed. I think this could benefit by the titles being looked at and all of the captions, because some of them could do with reworking.  Additionally, where can users view the rest of the descriptions that are longer and end with a … ? Usually that’s meant to signify there’s a “read more” button, but I’m not seeing where users can view the rest of this. I also think if it cannot appear on that main page, it is too long. But as of now, I can’t even find the rest of the description. I’d think it would be under Lesson Info but it isn’t.

This is just some of what I noticed. I could give it another look down the line to tackle more. But this is a good start.

**Comment by kmuir1991** [comment]
@seanlip based on @stepmalt 's comments above, does it make sense to assign a UXD to this so we can generate some mocks?

**Comment by seanlip** [comment]
Hi @stepmalt, I think you might be commenting on the wrong thing ... the lessons in the community library are created by the community and we don't have control over their titles and descriptions.

This issue is specifically meant for critique on the callout displayed in the image in the original issue description. Do you have any feedback on that in particular? Thanks!

**Comment by tydesignit** [comment]
**Initial revision suggestion:**

"Check out the first comprehensive course in the all-new Oppia Classroom! Curated lessons examined by educators where learners master math skills from various topics."

![image](https://github.com/oppia/design-team/assets/47770536/7c060882-64e2-4b92-8541-364e3cfa08e4)

^ Revision may require some padding adjustments as I shortened the length of the copy in the second sentence.

Also, I believe _course_ is the correct term in this context. I referred to the Content Style Guide in this section ([Writing about Oppia](https://docs.google.com/document/d/14FfNXmJ0lWXWz_PKKY2OG-FraiKlA2Y4m1UWMUsYaBw/edit#heading=h.nqj9q54tsamn)).

**Comment by stepmalt** [comment]
Check out the first comprehensive course in the all-new Oppia Classroom! These curated lessons are examined by educators, giving users support to master math skills from various topics.

Check out the first comprehensive course in the all-new Oppia Classroom! These curated lessons, where learners master math skills, are examined by educators.

**Comment by jikim322** [comment]
@kmuir1991 @tydesignit @stepmalt I wonder if we should discuss this during our meeting on Wednesday. Given the background info that Sean gave us about the difference between Oppia Classroom lessons and the community library, it could be confusing to include this callout so prominently on this page. I think it might make more sense to give a description of the community library and then have a less emphasized link to the Classroom.

**Comment by stepmalt** [comment]
@jikim322 I agree! That does seem confusing. Let's discuss during our meeting!

**Comment by kmuir1991** [comment]
@seanlip assigning this one to Ji, who thinks a bit of an audit of this page would be beneficial. Just a heads up

**Comment by jikim322** [comment]
@seanlip @kmuir1991 I've reviewed the current community page and made some notes/have some questions here: [Figma file](https://www.figma.com/file/R1ar93TAKjQpbMTlbFJ6YW/Oppia-Community-Library-page?type=design&node-id=9%3A2&mode=design&t=70GVoAKFv8cxNgKU-1)

 Would love feedback/thoughts!

**Comment by kmuir1991** [comment]
Thanks @jikim322 ! Left a few comments, but overall think your suggestions are great and make sense given the direction we're moving with our classroom + community library content.

**Comment by seanlip** [comment]
Thanks @jikim322, left some notes too!

**Comment by jikim322** [comment]
@seanlip @kmuir1991 Made some updates to the page based on feedback!

**Comment by davishedrick** [comment]
@seanlip @jikim322 UXW is good to go. Added suggestions to the Figma file in this comment thread.
https://www.figma.com/design/R1ar93TAKjQpbMTlbFJ6YW?node-id=9-2#689013076

**Comment by davishedrick** [comment]
Hi @jikim322 Adding a comment here for visibility. Would like confirmation on what screen is the final screen that we are handing off to developers. This UXW request involves the callout, but I noticed that in some updated screens in the figma mocks that the callout is missing. Please confirm:

1. Whether we are using the callout in the final screens
2. Whether there are any other UXW requests for this screen

Thanks!

**Comment by seanlip** [comment]
@davishedrick FYI I don't think @jikim322 is active on the team any more. Do you want to make the call for what we should do here, based on the current website?

**Comment by davishedrick** [comment]
@seanlip  I'm happy to take a second detailed look at the pages to see if there is any additional UXW need, but not sure how soon you want to push these updates. 

Just let me know which route you want to go.

**Comment by seanlip** [comment]
Hi @davishedrick, no strong hurry. Feel free to take another look -- thanks!

**Comment by seanlip** [comment]
This is obsolete -- the community library page was updated as part of "Multiple Classrooms" in #109. Closing as not required.
